### ISSUE TITLE
Broken syntax highlighting inside strings with slashes

### ISSUE THREAD

**Issue by mathrick** [defunkt/coffee-mode#299]
The following snippet breaks the highlighter:

```coffee
sLnk = prompt 'Write the URL here', 'http:\/\/'
if (sLnk&&sLnk != '' && sLnk != 'http://')
  execCommand 'createLink', false, sLnk
```

It's similar to what plagued earlier versions most of the time when slashes appeared inside string literals. I haven't seen this problem in a long time, but this reliably breaks things for me. The second line without the first is highlighted properly.

![screenshot from 2015-02-26 14 40 28](https://cloud.githubusercontent.com/assets/179487/6392986/6c80b61a-bdc6-11e4-9716-216cb8f5fa48.png)
![screenshot from 2015-02-26 14 48 47](https://cloud.githubusercontent.com/assets/179487/6393016/a7cb9f8c-bdc6-11e4-9836-4e0eac0cf433.png)

**Comment by syohex** [comment]
Thanks for report.

I fixed this issue at #300. Screenshot is here.

![coffee-mode](https://cloud.githubusercontent.com/assets/554281/6397172/4162a3fc-be26-11e4-95a9-d42abf318db7.png)

Please check latest version.

**Comment by mathrick** [comment]
Great, thank you. I can confirm it works.
